### PR TITLE
updated models for profile and pool

### PR DESCRIPTION
**Pull request by Pushpalatha555**

### CODE REVIEW

File: pkg/common/constants.go [hunk 1]
```diff
@@ -57,7 +57,9 @@ const (
 	RouterBgpNeighborPath          = "bgp-neighbors"
 	NetworkEdgeClusterPath         = "edge-clusters"
 	// Whoami Path
-	WhoamiPath = "whoami"
+	WhoamiPath            = "whoami"
+	LBPoolMemberGroupPath = "networks/servers/1/groups"
```
Comment (manjunath-batakurki) [inline pkg/common/constants.go:61]: this is not the correct, u can hardcode it like this

**Review by manjunath-batakurki — COMMENTED** [review]
minor comment, rest looks good

**Review by manjunath-batakurki — APPROVED** [review]
LGTM

File: pkg/common/constants.go (hunk 1 above)
Comment (manjunath-batakurki) [reply pkg/common/constants.go:61]: this will be taken up in the subsequent review. Approving it to unblock for now

**Review by manjunath-batakurki — APPROVED** [review]
LGTM

**Review by manjunath-batakurki — APPROVED** [review]
LGTM

**Review by manjunath-batakurki — DISMISSED** [review]

**Review by manjunath-batakurki — APPROVED** [review]
LGTM

**Review by gandharvas — APPROVED** [review]

**Pull request merged**